### DISCUSSION TITLE
Failed to submit process to grid scheduler for execution

### DISCUSSION THREAD

**Discussion by sqwwww** [nextflow-io/nextflow#2718] [Q&A]
Hi! 
I got an error when submit my job to qsub system. It said "Failed to submit process to grid scheduler for execution", and the log file is showed below. But when I run "qsub -N nf-getSelectedR .command.run" in my login node, it worked successfully. I guess it maybe caused by the difference between login node and the cpu node. could you please give me some suggestions about the problem?
[Mhap1.log](https://github.com/nextflow-io/nextflow/files/8238178/Mhap1.log)

**Comment by pditommaso** [comment]
I'd suggest asking for support to your cluster admins
